<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RFIDInputDialogClass</class>
 <widget class="QDialog" name="RFIDInputDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RFIDInputDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Renndaten</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="StarttimeLabel">
        <property name="whatsThis">
         <string>Startzeit</string>
        </property>
        <property name="text">
         <string>&lt;Starttime&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="EndtimeLabel">
        <property name="whatsThis">
         <string>Endzeit</string>
        </property>
        <property name="text">
         <string>&lt;Endtime&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="FahrzeitLabel">
        <property name="whatsThis">
         <string>Fahrzeit</string>
        </property>
        <property name="text">
         <string>&lt;Racetime&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="rfidsnrLabel">
        <property name="whatsThis">
         <string>Seriennummer der RFID-Karte</string>
        </property>
        <property name="text">
         <string>&lt;RFID_SNR&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="StreckennameLabel">
        <property name="text">
         <string>&lt;StreckenName&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="StreckenTypLabel">
        <property name="text">
         <string>&lt;SteckenTyp&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Kategorie:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="KategoriecomboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Kontaktdaten</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="whatsThis">
         <string>Name des Teilnehmers</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Vorname:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="2">
       <widget class="QLineEdit" name="vornameLineEdit">
        <property name="whatsThis">
         <string>Vorname des Teilnehmers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Adresse:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="adresseLineEdit">
        <property name="whatsThis">
         <string>Adresse des Teilnehmers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QRadioButton" name="maenlichRadioButton">
        <property name="text">
         <string>Männlich</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QRadioButton" name="weiblichRadioButton">
        <property name="text">
         <string>Weiblich</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>PLZ:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="plzLineEdit">
        <property name="whatsThis">
         <string>Postleitzahl des Teilnehmers</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Wohnort:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" colspan="2">
       <widget class="QLineEdit" name="ortLineEdit">
        <property name="whatsThis">
         <string>Wohnort des Teilnehmers</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nationalität:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="nationalitaetComboBox">
        <property name="whatsThis">
         <string>Liste der Nationalitäten</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Jahrgang:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Telefon:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="telephonLineEdit">
        <property name="whatsThis">
         <string>Telefon des Teilnehmers</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Mobile:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4" colspan="2">
       <widget class="QLineEdit" name="mobileLineEdit">
        <property name="whatsThis">
         <string>Mobile des Teilnehmers</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="emailLineEdit">
        <property name="whatsThis">
         <string>Email des Teilnehmers</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Team / Club:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4" colspan="2">
       <widget class="QLineEdit" name="clubLineEdit">
        <property name="whatsThis">
         <string>Team / Club des Teilnehmers</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="2">
       <widget class="QSpinBox" name="jahrgangSpinBox">
        <property name="minimum">
         <number>1900</number>
        </property>
        <property name="maximum">
         <number>2050</number>
        </property>
        <property name="value">
         <number>1980</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="autoCheckBox">
       <property name="text">
        <string>automatisches Vervollständigen (Standardmässig aktiviert)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>234</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="whatsThis">
          <string>Mit Ok werden die Information zur Datenbank hinzugefügt</string>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="canButton">
         <property name="whatsThis">
          <string>Verlässt diesen Dialog ohne Änderungen</string>
         </property>
         <property name="text">
          <string>Abbrechen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
